waspc/src/Wasp/Generator/Job/IO/PrefixedWriter.hs

```diff
@@ -172,8 +172,8 @@ makeJobMessagePrefix jobMsg =
   (T.pack . buildPrefix . concat)
     [ [("[", jobStyles)],
       [(jobName, jobStyles)],
-      styledFlags,
-      [("]", jobStyles)]
+      [("]", jobStyles)],
+      styledFlags
```

Comment:
What happened here, why this change? If I am correct, this results in having `[Wasp]!` instead of `[Wasp!]`?

Also, I see you added padding below to names.

I am not sure if I like these changes or not -> previous choices for not having padding and having this exclamation mark inside were not made randomly, so I would like to discuss this. But let's not do this inside of this PR then, you can do this when you will go about improving CLI as you said you would like to do.

Reply:
Makes sense, I'll move it.